### PR TITLE
service details: add offers & purchases tabs

### PR DESCRIPTION
**Pull request by ilgooz**

### CODE REVIEW

**Comment by NicolasMahe** [comment]
Do you have a example with data? None of the services has offers and purchases for now...

**Comment by ilgooz** [comment]
@NicolasMahe I've tested it with fake data. Let's merge this one and when we have the real data we can check to see if something is not pleasant. If so, we can then improve.

File: components/ServiceDetails/Sections/Offers.vue [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      <el-table-column
+        prop="duration"
+        label="Duration">
+      </el-table-column>
+      <el-table-column
+        prop="status"
+        label="Status">
+          <template slot-scope="scope">
+            <div class="status" :title="scope.row.status ? 'active' : 'passive'">
+              <status-indicator :positive="scope.row.status"></status-indicator>
```
Comment (NicolasMahe) [inline components/ServiceDetails/Sections/Offers.vue]: The property in the offer is `active` and not `status`.
I added an offer and disabled it but i didn't see any change in the ui.
https://deploy-preview-4--mesg-marketplace.netlify.com/services/0x6b7102dbfeb4332cbbdd5684f43592f5b91203037a1abc183d52dccc903c71ae#offers

Comment (ilgooz) [reply components/ServiceDetails/Sections/Offers.vue]: done

**Review by NicolasMahe — DISMISSED** [review]

File: components/ServiceDetails/Sections/Offers.vue [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      :data="offers"
+      style="width: 100%">
+      <el-table-column
+        prop="index"
+        label="Index"
+        width="180">
+      </el-table-column>
+      <el-table-column
+        prop="price"
+        label="Price">
```
Comment (antho1404) [inline components/ServiceDetails/Sections/Offers.vue]:
```suggestion
        label="Price (MESG Token)">
```

File: components/ServiceDetails/Sections/Offers.vue [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      </el-table-column>
+      <el-table-column
+        prop="duration"
+        label="Duration">
```
Comment (antho1404) [inline components/ServiceDetails/Sections/Offers.vue]:
```suggestion
        label="Duration (seconds)">
```

**Review by antho1404 — COMMENTED** [review]
Can we just add the units on the headers to know what we are talking about

File: components/ServiceDetails/Sections/Offers.vue (hunk 3 above)
Comment (ilgooz) [reply components/ServiceDetails/Sections/Offers.vue]: done

File: components/ServiceDetails/Sections/Offers.vue (hunk 2 above)
Comment (ilgooz) [reply components/ServiceDetails/Sections/Offers.vue]: done

**Review by NicolasMahe — APPROVED** [review]

**Pull request merged**